src/components/PreviewPane/PreviewPane.js

```diff
@@ -28,25 +29,48 @@ export default class PreviewPane extends React.Component {
     if (authorField) this.inferedFields[authorField] = INFERABLE_FIELDS.author;
   }
 
+  _getWidget = (field, value, props) => {
```

Comment:
Please, don't use `_` to start a name function. I know we have it in other places, but I'd like to stop doing that.